protocols/relay/examples/relay_v2.rs

```diff
@@ -25,12 +25,13 @@ use futures::stream::StreamExt;
 use libp2p::core::upgrade;
 use libp2p::identify::{Identify, IdentifyConfig, IdentifyEvent};
 use libp2p::multiaddr::Protocol;
+use libp2p::ping;
 use libp2p::relay::v2::relay::{self, Relay};
 use libp2p::swarm::{Swarm, SwarmEvent};
-use libp2p::tcp::TcpTransport;
```

Comment:
unrelated with this PR, should we setup a `rustfmt.toml` and configure [`imports_granularity`](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#imports_granularity) to maintain import consistency across the repo?

Reply:
That would be great but unfortunately, the setting is not yet stable so we can't use it.

You can hack around that but defining an alias same as we do for `clippy`. See [here](https://github.com/Restioson/xtra/blob/master/.cargo/config.toml) for example.

I'd be in favor of that :)